org/org.yaml

```diff
@@ -277,3 +277,399 @@ orgs:
         privacy: closed
         repos:
           triggers: write
+      website.maintainers:
+        description: the website maintainers
+        maintainers:
+        - bobcatfish
+        members:
+        - bobcatfish
+        - dlorenc
+        - vdemeester
+        - kimsterv
+        - abayer
+        - lucperkins
+        - afrittoli
+        - skaegi
+        - AlanGreene
+        - jjasghar
+        - popcor255
+        privacy: closed
+        repos:
+          website: write
+      core.collaborators:
+        description: The core collaborators
+        maintainers: !!set
+          pritidesai:
```

Comment:
uhm, names should be a list, not a dict

Reply:
Apparently sets are rendered by YAML as dicts with no value, which makes sense :)
Fixed that now.